Octokit/Clients/RepositoriesClient.cs

```diff
@@ -116,6 +116,12 @@ public Task<Repository> Create(string organizationLogin, NewRepository newReposi
                 {
                     throw new PrivateRepositoryQuotaExceededException(e);
                 }
+                
+                if (errorMessage != null && errorMessage.EndsWith("is an unknown gitignore template.", StringComparison.OrdinalIgnoreCase))
```

Comment:
I might have been too eager with this, given you need `EndsWith` to verify this. Disregard.